internal/gengapic/example.go

```diff
@@ -71,6 +71,24 @@ func (g *generator) exampleMethod(pkgName, servName string, m *descriptor.Method
 	g.imports[inSpec] = true
 
 	p("func Example%sClient_%s() {", servName, m.GetName())
+
+	pf, err := g.pagingField(m)
+	if err != nil {
+		return err
+	}
+
+	if *m.OutputType != emptyType {
+		p("// import \"%s\" %s", inSpec.Path, inSpec.Name)
```

Comment:
```suggestion
		p(`// import %s "%s"`, inSpec.Name, inSpec.Path)
```

Reply:
Thanks! committed it with `.want` file changes